extensions/2.x_official_extensions/gpc.md

```diff
@@ -0,0 +1,38 @@
+# Global Privacy Control
+
+Issue: [#98](https://github.com/InteractiveAdvertisingBureau/openrtb/issues/98)
+
+Sponsor: Magnite
+
+## Objective
+
+The purpose of this field is to support propagating the Global Privacy Control signal that originates in the HTTP headers from a web client, in order for all participants to have access to the same information on the privacy state of a user in regard to applicable regulations.
+
+## Request Changes
+
+<table>
+ <tr>
+  <th>Content Object</th>
+  <th>Type</th>
+  <th>Example values</th>
+  <th>Description</th>
+ </tr>
+ <tr>
+  <td>regs.ext.gpc</td>
+  <td>string</td>
+  <td>1</td>
+  <td>This value should exactly replicate the value in the Global Privacy Control signal setting received from the upstream request. Where that was an HTTP GET request, it is the contents of the `Sec-GPC` HTTP header. Where that was an OpenRTB request, it is the contents of the equivalent field.</td>
```

Comment:
One might leave it at "HTTP request," since the header should be sent on POST &c..

Reply:
I will rephrase.

Reply:
>Where that was an OpenRTB request, it is the contents of the equivalent field.

Since the only 'equivalent' field in an OpenRTB request would be _this_ field (correct?), how about, 

>Where that was an OpenRTB request, it is the contents of this field.

Also, values that are NOT from either the straight-from-device `Sec-GPC` or 'the equivalent' OpenRTB field are unspecified. If those two are all that we want this to carry, then that should be explicit,